app/frontend/src/features/connections/friends/useCancelFriendRequest.ts

```diff
@@ -28,7 +28,7 @@ export default function useCancelFriendRequest() {
         setMutationError("");
       },
       onSuccess: (_, { userId }) => {
-        queryClient.invalidateQueries("friendRequestsSent");
+        queryClient.invalidateQueries(["friendRequests", {"type":"sent"}]);
```

Comment:
Can you update this to use the `friendRequestKey` helper function in `src/queryKeys.ts` instead please? We are gradually moving towards centralising all of the query keys we use across the app so this type of mistakes will happen less often.

Reply:
Sure, just pushed.
Missed this file :-)